content/post/2022-05-28-cran-updates/index.en.Rmd

```diff
@@ -0,0 +1,2164 @@
+---
+title: CRAN updates
+author: Lluís Revilla Sancho
+date: '`r Sys.Date()`'
+slug: cran-updates
+categories:
+  - CRAN
+  - r
+tags:
+  - R
+  - packages
+authors: []
+description: 'Exploration of the meta archives of CRAN'
+draft: no
+editor_options:
+  chunk_output_type: console
+featured: no
+image:
+  caption: ''
+  focal_point: ''
+subtitle: "CRAN's package history"
```

Comment:
The title/subtitle could indicate a question?
The summary sounds very "explorative" so maybe some questions could help attract attention.

"Do you know when the first package was published on CRAN?" for instance